cli/cmd/secrets/create.go

```diff
@@ -30,6 +31,13 @@ const (
 )
 
 func (s *Create) Run(ctx *clicontext.CLIContext) error {
+	name := s.N_Name
+	if name != "" {
+		r := ctx.ParseID(fmt.Sprintf("secret/%s", name))
+		if _, err := ctx.Core.Secrets(r.Namespace).Get(r.Name, metav1.GetOptions{}); err == nil {
+			fmt.Printf("Warning: %s already exists\n", r)
```

Comment:
Shouldn't this be `err != nil` ? Also I would check against AlreadyExists error type, this could be a diff error.

Reply:
`err == nil` in this case because it returns no error if a secret with this name exists.

Reply:
Ahh, right.

Reply:
You're printing the v1.Secret type ? Why not just name.

Reply:
I was just printing similar to how we do it elsewhere. It ends up being the name with the type prepended:
```
$ ./bin/rio secret create -n githubtoken --docker
Warning: secret/githubtoken already exists
```